sqlglot/generator.py

```diff
@@ -716,13 +740,22 @@ def tablealias_sql(self, expression: exp.TableAlias) -> str:
         return f"{alias}{columns}"
 
     def bitstring_sql(self, expression: exp.BitString) -> str:
-        return self.sql(expression, "this")
+        this = self.sql(expression, "this")
+        if self.bit_start or self.bit_end:
```

Comment:
why are you checking of self.bit_end? can bit_end ever be empty? can bit start be empty?

Reply:
> can bit_end ever be empty

That's correct, for example in Clickhouse we have `BIT_STRINGS = [("0b", "")]`, so `bit_end` would be empty.

> can bit start be empty?

Not with the current formats allowed, so I can simplify these checks to `if self.bit_start`.

Reply:
My initial reasoning was that by checking `bit_start` it'd be more robust (e.g. for future cases where we might get literals that only have an ending delimiter), but on second thought we can probably postpone it until there's an actual issue.